wp-content/themes/goonj-crm/functions.php

```diff
@@ -159,6 +157,11 @@ function goonj_handle_user_identification_form() {
 
         $contact = $foundContacts[0];
 
+        // Check if the contact is a volunteer
+        if ( empty( $contact['contact_sub_type'] ) ) {
```

Comment:
This is an incomplete check. In the later stages, we will have `Individual` with other subtypes as well like `Fellow` or `Intern`. So we should also put a check when the **Individual has a subtype but not Volunteer**
```suggestion
        if ( empty( $contact['contact_sub_type'] ) || $contact['contact_sub_type'] !== 'Volunteer'  ) {
```